packages/pv-create-component/.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
i think the editorconfig should be in the repo-root

Reply:
in general yes, but i think on the long run it would be more helpful if each package could be maintained: edited, build, run, tested separately.